ksql-engine/src/main/java/io/confluent/ksql/function/InternalFunctionRegistry.java

```diff
@@ -81,12 +83,18 @@ private void init() {
     KsqlFunction substring = new KsqlFunction(Schema.STRING_SCHEMA, Arrays.asList(Schema
                                                                                   .STRING_SCHEMA,
                                                                               Schema
-                                                                                  .INT32_SCHEMA,
+                                                                                  .INT64_SCHEMA,
```

Comment:
This uses java's `substring` so the args should be int.

Reply:
There is no `IntegerLiteral` hence the use of Long. If you look at the `SubstringKudf` you'll see it expects `Long` arguments. Anyway, i'll add an `IntegerLiteral`